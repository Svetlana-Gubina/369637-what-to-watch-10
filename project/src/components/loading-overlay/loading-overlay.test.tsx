import React from 'react';
import { render } from '@testing-library/react';
import LoadingOverlay from './loading-overlay';

describe('LoadingOverlay component test', () => {
  test('should render component correctly', () => {
    const { container } = render(<LoadingOverlay />);

    expect(container).toMatchSnapshot();
  });
});
