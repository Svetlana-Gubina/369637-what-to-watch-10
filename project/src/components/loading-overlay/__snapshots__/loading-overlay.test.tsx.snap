// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingOverlay component test should render 1`] = `
<div>
  <div
    style="color: wheat; background: rgba(0, 0, 0, 0.5); width: 100vw; height: 100vh; margin: 0px; padding: 0px; position: fixed; top: 0px; right: 0px; z-index: 100; display: flex; justify-content: center; align-items: center; font-size: 25px; text-align: center;"
  >
    <p>
      Loading...
    </p>
  </div>
</div>
`;
